docs/source/upgrade_notes.rst

```diff
@@ -3,6 +3,13 @@ Upgrade Notes
 
 |st2| In Development
 --------------------
+* New ``abandoned`` action execution status has been introduced. This is effectively a failure
```

Comment:
An execution is set to ``abandoned`` state when an action runner running it is killed via TERM.